### ISSUE TITLE
Csu won't build on HEAD

### ISSUE THREAD

**Issue by joelteon** [copumpkin/nixpkgs#54]
```
these derivations will be built:
  /nix/store/f5536g7z260a61x9iqwrckzhnplbyzq8-unifdef-2.6.drv
  /nix/store/fri3yf8qfzq0nfxwhqd69s76jr0mva02-ed-1.10.drv
  /nix/store/ii9cjch9yjaazrxgs2kcj641ajw7sk4s-Libc-825.40.1.drv
  /nix/store/048l1b9s9268m5hf32ld09wiyf89j5id-Libc-997.90.3.drv
  /nix/store/dvqmsfwyxxbrj79qgvng1h41vpkwirpd-perl-5.20.1.drv
  /nix/store/0rzdfbmyh2k6myayjr4dj8006qqm44ff-groff-1.22.3.drv
  /nix/store/9mqbs21in53z39283alqga5vppp932cw-gnum4-1.4.17.drv
  /nix/store/89igp4wbmdjd7vwam3rk89fynq1j9aig-bison-3.0.2.drv
  /nix/store/jpjylb5chi7nbl01fb6pbw1rvpjv25pi-flex-2.5.39.drv
  /nix/store/dcb4rzzd9nxlvgjkxfw1glcpymq177i8-bootstrap_cmds-86.drv
  /nix/store/16f2kw0bwm5b1sqqkc87n9zbq8l79mkx-xnu-2422.115.4.drv
  /nix/store/1h8k62bzf69lynsjsil6sjmczbm9q0gs-Csu-79.drv
  /nix/store/1zjgir9dd7y7kgv32lpdkganc7h595sy-Libinfo-449.1.3.drv
  /nix/store/7byyb3kkp8alkdzn0q2619rx1caa4474-libpthread-105.1.4.drv
  /nix/store/81aaxsiza8a3g73plyjkcr8rbvfhk20d-autoconf-2.69.drv
  /nix/store/c5s14bbkysjlvzyasrb0py63a67w4f63-openssl-1.0.1k.drv
  /nix/store/j4mp7iiwpi6bvfbbqxgg7pcblghwnlqn-objc4-551.1.drv
  /nix/store/lpqmbyqrvn6qrh4v6pdfjmv68pk00d3k-hook.drv
  /nix/store/jyzgsh9b1wb8s3na30bp5s3gh0c2jk6k-automake-1.12.6.drv
  /nix/store/1iixfg3g4ksfhr4sq6w2ngkhpajmwrm8-zlib-1.2.8.drv
  /nix/store/680q1npx36dgknz1l2n5cpbwsr3zcz6x-bzip2-1.0.6.drv
  /nix/store/9ykrj49lhwia14cfszrbd3v38c083yjp-ps-153.drv
  /nix/store/j0n9i3fmvynzww21cbrl63ifdjd1cvfl-ncurses-5.9.drv
  /nix/store/ndcbng2l9bfri58yil72pidkd0whb7sx-libssh2-1.4.3.drv
  /nix/store/kcr655sd59ia02yf852pvx7vinp5zwqp-curl-7.39.0.drv
  /nix/store/1axgxp7ha98lxfzkg6xrzry5ap8ma836-hook.drv
  /nix/store/g0vk0q7nd5fp194sg4ghj6gnsywnhm8n-libxml2-2.9.2.drv
  /nix/store/fyvvbbfz605sb8dyhn3v0j8zj022aj79-gmp-5.1.3.drv
  /nix/store/5i1xpbi7kb2549j9ql91bpb60d4h14a1-coreutils-8.23.drv
  /nix/store/shxava0kd6529figi2h31qnk24b6wj6i-libiconv-1.14.drv
  /nix/store/pb4vzblwkrg2j86vq0ryp01vkah38v6n-gettext-0.18.2.drv
  /nix/store/xxg53v33fmjqwmxyf1vlwaj7xgqh2blc-sharutils-4.11.1.drv
  /nix/store/y8859wh0b108cp44pnnbgi7mbfvpkl1y-libarchive-3.1.2.drv
  /nix/store/ykg0dwyz37lvdgjzib9kpkq0plry1f0n-expat-2.1.0.drv
  /nix/store/dpzdjvfbc0dyzlsi420sn21dhsb1pxxq-cmake-2.8.12.2.drv
  /nix/store/vq6dvjp0r9q2jif1ljv0xdw2km0fm94d-libc++abi-3.5.0.drv
  /nix/store/vx3la5iv4c3pjadk186rx0p51ljxf7kh-libtool-2.4.2.drv
  /nix/store/f4n4jw03ma6fq93hwk45gdq17f05faph-cctools-port-862.drv
  /nix/store/h187qrs5pnh3cgvg5idkm6pld4fr6vqg-cpio-2.11.drv
  /nix/store/4b90kn74kxn4gidybkq9hhj73mdnv9la-IOKit-907.100.13.drv
  /nix/store/hk3bxpckpbssyp96p7b6vsxz8niskhjj-Security-55471.14.18.drv
  /nix/store/x6mkz8q6xhlg4bfqwzccw01q4k98fxyc-ppp-727.90.1.drv
  /nix/store/3wqvrpqgvqs2znca437n104zrysqhiaw-configd-453.19.drv
  /nix/store/k5az758yc5xr0jhnzw1xyrahsfkbhdwh-libresolv-54.drv
  /nix/store/h8qgr44f267x0rvpgw4v1wa09nh2fkcp-Libsystem-1197.1.1.drv
  /nix/store/2qi05dwskxq12yq06g3x5bwsmzz2anh5-clang-wrapper-9.9.9.drv
  /nix/store/mz1vaphs7mrw9a382vkjv00fs2ag42bs-hook.drv
  /nix/store/75ii909ca5qj1ab5l7gbkqxzcrp89rvl-icu4c-53.1.drv
  /nix/store/pp22m0r8dap5vdil1phr2h1m3m2wcyms-CF-855.17.drv
  /nix/store/qhbmxr9b4lza4d1r09csfkbcbi2fjmgi-libc++-3.5.0.drv
  /nix/store/n14wfsp0sv5m8pih9sh9np9jx2ra6h7r-stdenv-darwin-boot-2.drv
  /nix/store/16n4vw0c6ccmpjhmjbji0zpm4xnvq1x1-bash-4.3-p33.drv
  /nix/store/ci63drmvnbv5nm5fdz9kxckhc9gvblaf-CF-855.17.drv
  /nix/store/r2q5hspvr1k2myv02hn2d1n8bix2swj0-libc++abi-3.5.0.drv
  /nix/store/va19cgixpfnkil72h6bk1vsf3niz84kb-clang-wrapper-9.9.9.drv
  /nix/store/k3jp0mpva3pzq1ik9vpkvs0fhzgjyf9n-hook.drv
  /nix/store/wh9rb0blym3kxj7h0fbq6b20ns8x4fsq-libc++-3.5.0.drv
  /nix/store/fbmyxsdfmmgvxvr8aw4caq1sjdi63z50-stdenv-darwin-boot-3.drv
  /nix/store/wh7v02pym4syrdsfv1jp6khdbvl6k1di-objc4-551.1.drv
  /nix/store/7q50idjmb7b2xwsjxvgzxn027cwfqzlj-cctools-port-862.drv
  /nix/store/8z3hgjvr8ghw3b80y1l6gh8ffnys2qa2-libffi-3.0.13.drv
  /nix/store/a9nr8i35l42m0zzrkbqrra1l2hbnafw4-zlib-1.2.8.drv
  /nix/store/v3z5mc6prn1mqb7dplnqhxhk7wbnv694-hook.drv
  /nix/store/92xdz62710ff1ivnbszpk0w895ar951y-libxml2-2.9.2.drv
  /nix/store/9i9vaj2r2acnc3l5grfb5cbzbr9hlz10-ncurses-5.9.drv
  /nix/store/lcw1qmrzzz089rwmja7kj9jjf4py8i23-python-2.7.9.drv
  /nix/store/1gyikdg2sgqq5l32yyp3cg0y1mnd7dyy-llvm-3.5.0.drv
  /nix/store/9d7b494nxwpyinik1vbngqr186j6z0wx-gmp-5.1.3.drv
  /nix/store/1yyffv13fly0r2mkjkwxi7x9gx77j02a-coreutils-8.23.drv
  /nix/store/2fkrm24ps88afvrsf1y6ij8l1f01jyl6-libiconv-1.14.drv
  /nix/store/2ji1a616ph93zad42n4dh64sp52rfj19-xz-5.2.0.drv
  /nix/store/58f94c2iw9aqw1l9hhw1d6i20cb9pm83-findutils-4.4.2.drv
  /nix/store/5j025wkjk5dp52p2flclv6dk130pq1y6-gnumake-3.82.drv
  /nix/store/60yi2447q0xwd15pwx054wakqihh20z6-diffutils-3.3.drv
  /nix/store/am8n5h4a5hvdny8khxw46yalnnckg5pk-gnutar-1.27.1.drv
  /nix/store/dmah5yp2cslnylhzjzvsizib4zk3ciz5-hook.drv
  /nix/store/c3d2a5waxcirq28f7cfp1yzk6ivy2kxd-icu4c-53.1.drv
  /nix/store/vdzjan4y22zs4yj9sn01l75bcvzr76w8-libedit-20140620-3.1.drv
  /nix/store/mjrjwbdgmb1xx9klc9sw3czq9nj0ssk0-clang-3.5.0.drv
  /nix/store/d2bmg1nwc51137hcwdwpabxwgzw80v5i-clang-wrapper-3.5.0.drv
  /nix/store/f3lkggw9jarzs4s6p9d5lpawlsqbjh52-ed-1.10.drv
  /nix/store/fb8xqi2m5xfylcym7k555b13hnra2c9l-patch-2.7.1.drv
  /nix/store/h3790f1fxakwx88a5rp6rbn1ri327b5n-gnused-4.2.2.drv
  /nix/store/j06cffys4sqc4w2vi07lfrsldmvvq16h-pcre-8.36.drv
  /nix/store/lp784rnxj6a1zcjjb2dv3j7v91ik5zvm-bzip2-1.0.6.drv
  /nix/store/v24pai5xhxbirh9lw8qcdb9sss8ngmbr-gzip-1.6.drv
  /nix/store/x6g4mgd7frcwp8khzxigd2s04gbnqr6h-libsigsegv-2.10.drv
  /nix/store/v5yh7j6pl3bq3qjph649ngdb6vswycl0-gawk-4.1.0.drv
  /nix/store/v625wyyn9s98hc9jzs3pc57s6d0dah2m-gnugrep-2.20.drv
  /nix/store/ylbl9nxz8y7708xylbrl6ambjdkgv5xm-CF-855.17.drv
  /nix/store/xq44gkgx94kidh4a4wiqlvagda66dp50-stdenv-darwin.drv
  /nix/store/51iwfiwr8lyi1vqg70r2r1brgja53rpi-launchd-842.92.1.drv
  /nix/store/plir0058428fvrx1hyxnic5fh94sv79l-MacOSX10.9.sdk.drv
  /nix/store/9h7743mhy5nzxrss5dz181mry98wc6mk-perl-5.20.1.drv
  /nix/store/crfyjs26vqykwbxqjph46ggv81f3jjzb-unifdef-2.6.drv
  /nix/store/ri571l64chk6wnzid61ysfib2p5lld4z-gnum4-1.4.17.drv
  /nix/store/nssr5zhfbvx6rsny3qlmccdqm3n2949p-bison-3.0.2.drv
  /nix/store/hm6n8k0ig4c1bp08vgj16xb949rhs8l3-flex-2.5.39.drv
  /nix/store/yvhgwlf4m1m0iir36jlv0rmxf24bipz9-bootstrap_cmds-86.drv
  /nix/store/63mmhwgprq64ya28cqpiqs6h5swf7q1i-xnu-2422.115.4.drv
  /nix/store/8pla3q1cg2lqgzvq631j4fd3agj8f5dq-eap8021x-180.drv
  /nix/store/8znkmf6rjy4pnyr7x1g6f10r4jg0b6fg-IOKit-907.100.13.drv
  /nix/store/ynmbkbzbkzf7yinqz2rzl6bx54ch1m6v-ppp-727.90.1.drv
  /nix/store/yq1z8icch1b7s063ziqsnzwdaijw84z1-Security-55471.14.18.drv
  /nix/store/86krqsmzvs63f73c6jd5ln3kfa0w9shd-configd-453.19.drv
  /nix/store/fpgqvy2s55pxdr453rgphyjh69jrizm1-openssl-1.0.1k.drv
  /nix/store/v08n03mqx6pgyy0n62d314y4n4627y1x-python-2.7.9.drv
  /nix/store/5mynjf55l9cg99m9hfpinn3m7lvpznz1-PrivateMacOSX10.9.sdk.drv
  /nix/store/f7xf11r18w2glsbaf133jznggknbgm5r-SecurityTool-55115.drv
building path(s) ‘/nix/store/bary8ahc13ycfvv5406wnz9z0r8wxr77-Csu-79’
unpacking sources
unpacking source archive /nix/store/ba6vvp3y28lkyhr9cy87fdcgqmq26ijj-Csu-79.tar.gz
source root is Csu-79
patching sources
configuring
no configure script, doing nothing
building
make flags: SHELL=/nix/store/04h34n8711qb4njn4fdj3l5spxm39gzq-bootstrap-tools/bin/sh    
Makefile:120: target `/lib' given more than once in the same rule.
building crt1.v1.o
clang -r  -Os -mmacosx-version-min=10.4 -mdynamic-no-pic -nostdlib -keep_private_externs start.s crt.c dyld_glue.s -o crt1.v1.o  -DCRT -DOLD_LIBSYSTEM_SUPPORT
building crt1.v2.o
clang -r  -Os -mmacosx-version-min=10.5 -nostdlib -keep_private_externs start.s crt.c dyld_glue.s -o crt1.v2.o  -DCRT
building crt1.v3.o
clang -r  -Os -mmacosx-version-min=10.6 -nostdlib -keep_private_externs start.s crt.c -o crt1.v3.o  -DADD_PROGRAM_VARS 
building gcrt1.o
clang -r  -Os -mmacosx-version-min=10.4 -nostdlib -keep_private_externs start.s crt.c dyld_glue.s -o gcrt1.o  -DGCRT  -DOLD_LIBSYSTEM_SUPPORT
building crt0.o
clang -r  -Os -static -nostdlib -keep_private_externs start.s crt.c -o crt0.o 
building dylib1.v1.o
clang -r  -Os -mmacosx-version-min=10.4  -nostdlib -keep_private_externs dyld_glue.s icplusplus.c -o dylib1.v1.o  -DCFM_GLUE
icplusplus.c:1:10: fatal error: 'stdlib.h' file not found
#include <stdlib.h>
         ^
1 error generated.
make: *** [dylib1.v1.o] Error 1
builder for ‘/nix/store/1h8k62bzf69lynsjsil6sjmczbm9q0gs-Csu-79.drv’ failed with exit code 2
cannot build derivation ‘/nix/store/h8qgr44f267x0rvpgw4v1wa09nh2fkcp-Libsystem-1197.1.1.drv’: 1 dependencies couldn't be built
building path(s) ‘/nix/store/qbc47yr5fc892lzdhv2fy0z81sfqq1v1-Security-55471.14.18’
killing process 87888
building path(s) ‘/nix/store/80ww285hcy2qjvwf30fdqzz5zba2b467-gnum4-1.4.17’
cannot build derivation ‘/nix/store/v08n03mqx6pgyy0n62d314y4n4627y1x-python-2.7.9.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/n14wfsp0sv5m8pih9sh9np9jx2ra6h7r-stdenv-darwin-boot-2.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/xq44gkgx94kidh4a4wiqlvagda66dp50-stdenv-darwin.drv’: 1 dependencies couldn't be built
killing process 87889
cannot build derivation ‘/nix/store/f7xf11r18w2glsbaf133jznggknbgm5r-SecurityTool-55115.drv’: 1 dependencies couldn't be built
error: build of ‘/nix/store/f7xf11r18w2glsbaf133jznggknbgm5r-SecurityTool-55115.drv’ failed
```

**Comment by copumpkin** [comment]
In general, please include git hash (HEAD means different things at different times) and invocation/chroot build status. As I mentioned on IRC, it's fine here, also on HEAD. I'm on a24e68ab567886f37efee75f5175bdcf63c5fb35.

**Comment by joelteon** [comment]
Whoops, that'll be a24e68a for me also

**Comment by copumpkin** [comment]
For anyone looking, the cause was a case-sensitivity issue. @joelteon runs a case-sensitive filesystem and I was sloppy when I changed the case of `Libsystem` recently.

**Comment by copumpkin** [comment]
Fixed by ade64608ec7f15de59f0040bdb5392e25cf13025
